lib/platform/azure/index.md

```diff
@@ -0,0 +1,5 @@
+# Azure DevOps and Azure DevOps Server
+
+## Features awaiting implementation
+
+- The `automergeStrategy` configuration option has not been implemented for this platform, and all values behave as if the value `auto` was used. Renovate will use the merge strategy configured in the Azure Repos repository itself, and this cannot be overridden yet
```

Comment:
```suggestion
- The `automergeStrategy` configuration option has not been implemented for this platform, and all values behave as if the value `auto` was used. Renovate will use the merge strategy configured in the Azure DevOps / Azure DevOps Server repository itself, and this cannot be overridden yet
```

Are we sure it's `Azure Repos` and not something else?

Reply:
Strictly speaking, yes. Azure Repos is the product included in the Azure DevOps suite of services (In the same way that Azure Boards is "Microsoft Jira", for instance). The product pages for [Azure DevOps](https://azure.microsoft.com/en-us/services/devops/) and [Azure DevOps Server](https://azure.microsoft.com/en-us/services/devops/server/) are what I'm basing this on.

However, if we've established a convention of referring to Azure Repos as "Azure DevOps" for the purposes of Renovate (which is based around repositories), I can appreciate if we'd like to make this change.

Reply:
Let's be technically correct then, and use Azure Repos as name. 😉